### PR TITLE
Feature/change token policy

### PR DESCRIPTION
**Pull request by jarcane**
the UI here is a bit rough

experimenting with maybe making it a separate page instead but still working out how to even do that

### CODE REVIEW

**Comment by neurogoo** [comment]
One option could be to overload the new token dialog to add new token to user or update it's policy if user is found in database

File: futurice-foundation/src/Futurice/Lomake.hs [hunk 1]
```diff
@@ -172,6 +172,9 @@ vHidden = VHidden
 vDynamic :: [(a, Text)] -> V a
 vDynamic = V Nothing
 
+vJustDynamic :: a -> [(a, Text)] -> V a
```
Comment (neurogoo) [inline futurice-foundation/src/Futurice/Lomake.hs]: Why did you need new field type?

Comment (jarcane) [reply futurice-foundation/src/Futurice/Lomake.hs]: I cannot seem to provide *both* a default entry, *and* a dynamic field type. `vDynamic` causes it to default to empty, and `vJust` only works with regular enums, so there's no way to have a default dynamic entry showing otherwise. Or at least if there is I couldn't figure out the means, but this does what was needed.

**Comment by jarcane** [comment]
Rewrote the token form to allow for upsert, seems to work great
